### DISCUSSION TITLE
How to update dependency everywhere?

### DISCUSSION THREAD

**Discussion by Tobjoern** [vercel/turborepo#5564] [Help] [answered]
We have a turborepo with various trpc apis.
The problem with trpc is, that both the client + server need to have the same version, for it to function properly,
thus we require the same version everywhere.

What is the best way to update trpc to be the same version in every package + app?

**Comment by joafeldmann** [comment, marked as answer]
You can either install your dependency to the workspace root or you can use pnpm to update trpc in all packages recursively: `pnpm up -r my-dep@some-version`

https://pnpm.io/cli/update#--recursive--r
